.gitignore

```diff
@@ -16,6 +24,14 @@ HasalServer.log.*
 stage/*
 docs/*
 
+# File types to ignore
+*.zip
+*.diff
+*.json
```

Comment:
The new Json settings will all store in configs folder...svrconfig.json will be removed after the new configs merged